### DISCUSSION TITLE
Composer Probleme

### DISCUSSION THREAD

**Discussion by niskia** [kimai/kimai#4316] [General]
Ich habe schon mehrmals versucht Kimai zu installieren (Auf einem Plesk Server) aber bin immer beim Composer hängen geblieben.

```
Executing script cache:clear [KO]
 [KO]
Script cache:clear returned with error code 255
!!  
!!   // Clearing the cache for the prod environment with debug false                
!!  
!!  PHP Fatal error:  Allowed memory size of 134217728 bytes exhausted (tried to allocate 655360 bytes) in /var/www/vhosts/doamine.de/worktime.domaine.de/kimai/vendor/twig/twig/src/NodeVisitor/SafeAnalysisNodeVisitor.php on line 71
!!  PHP Fatal error:  Allowed memory size of 134217728 bytes exhausted (tried to allocate 20480 bytes) in /var/www/vhosts/domaine.de/worktime.domaine.de/kimai/vendor/symfony/var-dumper/Dumper/CliDumper.php on line 421
```

Hat jemand eine Idee?

**Comment by kevinpapst** [comment]
10 seconds with your favorite search engine:
https://getcomposer.org/doc/articles/troubleshooting.md#memory-limit-errors
https://mikemadison.net/blog/2020/8/13/troubleshooting-composer-out-of-memory-errors

Or you simply RTFM
https://www.kimai.org/documentation/installation.html#memory-limit
